kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -48,35 +47,29 @@
 
 <script>
 
+  const constants = require('../../state/constants');
+  const getters = require('../../state/getters');
+
   module.exports = {
-    created() {
-      this.fetchFullContent(this.id);
-    },
+    mixins: [constants], // makes constants available in $options
```

Comment:
So, could we pass in this `pageMode` state as a prop from the `index.vue`? I feel like that fits better with the patterns we have discussed previously.

Also, that means we only have to import the constants at the top level.